### PR TITLE
pd-ctl: add scheduler config evict-leader-scheduler

### PR DESCRIPTION
**Pull request by glkappe**
This update is not comprehensive, please help modify and add

### What is changed, added or deleted? (Required)

Add `scheduler config evict-leader-scheduler` and its description.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: pd-control.md [hunk 1]
```diff
@@ -961,6 +961,7 @@ Encoding 格式示例：
 >> scheduler show                                 // 显示所有的 schedulers
 >> scheduler add grant-leader-scheduler 1         // 把 store 1 上的所有 Region 的 leader 调度到 store 1
 >> scheduler add evict-leader-scheduler 1         // 把 store 1 上的所有 Region 的 leader 从 store 1 调度出去
+>> scheduler config evict-leader-scheduler        // v4.0.4 起, 展示该调度具体在哪些 store 上
```
Comment (rleungx) [inline pd-control.md:964]:
```suggestion
>> scheduler config evict-leader-scheduler        // v4.0.0 起, 展示该调度具体在哪些 store 上
```

**Review by glkappe — COMMENTED** [review]
Add description of ```store-id```

**Review by rleungx — APPROVED** [review]

**Review by lhy1024 — APPROVED** [review]

File: pd-control.md [hunk 2]
```diff
@@ -961,9 +961,10 @@ Encoding 格式示例：
 >> scheduler show                                 // 显示所有的 schedulers
 >> scheduler add grant-leader-scheduler 1         // 把 store 1 上的所有 Region 的 leader 调度到 store 1
 >> scheduler add evict-leader-scheduler 1         // 把 store 1 上的所有 Region 的 leader 从 store 1 调度出去
+>> scheduler config evict-leader-scheduler        // v4.0.0 起, 展示该调度具体在哪些 store 上
```
Comment (TomShawn) [inline pd-control.md:964]:
```suggestion
>> scheduler config evict-leader-scheduler        // v4.0.0 起，展示该调度具体在哪些 store 上
```

File: pd-control.md [hunk 3]
```diff
@@ -961,9 +961,10 @@ Encoding 格式示例：
 >> scheduler show                                 // 显示所有的 schedulers
 >> scheduler add grant-leader-scheduler 1         // 把 store 1 上的所有 Region 的 leader 调度到 store 1
 >> scheduler add evict-leader-scheduler 1         // 把 store 1 上的所有 Region 的 leader 从 store 1 调度出去
+>> scheduler config evict-leader-scheduler        // v4.0.0 起，展示该调度具体在哪些 store 上
 >> scheduler add shuffle-leader-scheduler         // 随机交换不同 store 上的 leader
 >> scheduler add shuffle-region-scheduler         // 随机调度不同 store 上的 Region
->> scheduler remove grant-leader-scheduler-1      // 把对应的 scheduler 删掉
+>> scheduler remove grant-leader-scheduler-1      // 把对应的 scheduler 删掉，`-1` 对应 store-id
```
Comment (TomShawn) [inline pd-control.md:967]:
```suggestion
>> scheduler remove grant-leader-scheduler-1      // 把对应的 scheduler 删掉，`-1` 对应 store ID
```

File: pd-control.md [hunk 4]
```diff
@@ -961,9 +961,10 @@ Encoding 格式示例：
 >> scheduler show                                 // 显示所有的 schedulers
 >> scheduler add grant-leader-scheduler 1         // 把 store 1 上的所有 Region 的 leader 调度到 store 1
 >> scheduler add evict-leader-scheduler 1         // 把 store 1 上的所有 Region 的 leader 从 store 1 调度出去
+>> scheduler config evict-leader-scheduler        // v4.0.0 起，展示该调度具体在哪些 store 上
```
Comment (TomShawn) [inline pd-control.md:964]:
```suggestion
>> scheduler config evict-leader-scheduler        // v4.0.0 起，展示该调度器具体在哪些 store 上
```

File: pd-control.md [hunk 5]
```diff
@@ -961,9 +961,10 @@ Encoding 格式示例：
...
 >> scheduler add shuffle-leader-scheduler         // 随机交换不同 store 上的 leader
 >> scheduler add shuffle-region-scheduler         // 随机调度不同 store 上的 Region
->> scheduler remove grant-leader-scheduler-1      // 把对应的 scheduler 删掉
+>> scheduler remove grant-leader-scheduler-1      // 把对应的 scheduler 删掉，`-1` 对应 store ID
```
Comment (TomShawn) [inline pd-control.md:967]:
```suggestion
>> scheduler remove grant-leader-scheduler-1      // 把对应的调度器删掉，`-1` 对应 store ID
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**